perf.groovy

```diff
@@ -0,0 +1,32 @@
+// Import the utility functionality.
+
+import jobs.generation.*;
+
+def project = GithubProject
+def branch = GithubBranchName
+def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolderName(branch)
+
+
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        def newJob = job(Utilities.getFullJobName(project, "perf_${os}", isPR)) {
+            // Set the label.
+            steps {
+                    // Batch
+                    batchFile("set __TestIntermediateDir=int&&build.cmd release x64")
```

Comment:
Thanks for the review Jarret. 

Actually I split three steps on purpose to match the convention of existing coreclr windows Jenkins runs. For example, the run http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/debug_windows_nt_prtest/configure splits the build and test run steps into 6 different batch files, instead of put them into one one. Seems to me the coreclr Jenkins run preference is to split commands into batch files as many as possible, so the log is clearer on the configuration page, and each split command can be easily reused.